specs/language/statements.tex

```diff
@@ -1,2 +1,74 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The Attribute \textbf{[unroll]} and \textbf{[unroll(n)]} attribute qualifiers can be used to specify that a loop (\textbf{for}, \textbf{while} and \textbf{do while} loops) can be unrolled. This attribute qualifier can be used to specify full unrolling or partial unrolling by a specified amount. This is a compiler hint and the compiler may ignore this directive.
+
+n is the loop unrolling factor and must be a positive integral compile time constant expression. If n is not specified, the compiler determines the unrolling factor for the loop. This attribute is not compatible with the [loop] attribute.
+
+Note:  The \textbf{[unroll]} attribute  must appear immediately before the loop to be affected.
+
+\subsubsection{Unroll Examples}{Stmt.Attr.Unroll.Examples}
+\begin{itemize}
+\item  Tell the compiler to unroll the above \textbf{for} loop by a factor of 2.
```

Comment:
Wondering if the text here should be tweak to account for "This is a compiler hint and the compiler may ignore this directive.".

Reply:
I can change the woring to Requests or Suggests. We did talk about it being a hint earlier.